dummyserver/https_proxy.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+import sys
+from typing import Any, Dict
+
+import tornado.httpserver
+import tornado.ioloop
+import tornado.web
+
+from dummyserver.proxy import ProxyHandler
+from dummyserver.server import DEFAULT_CERTS, ssl_options_to_context
+
+
+def run_proxy(port: int, certs: Dict[str, Any] = DEFAULT_CERTS) -> None:
+    """
+    Run proxy on the specified port using the provided certs.
+
+    Example usage:
+
+    python -m dummyserver.proxy
```

Comment:
Is this supposed to be `python -m dummyserver.https_proxy`?

Reply:
oops, yes, let me fix this.